src/Search/Microsoft.Azure.Search/Customizations/Indexers/Models/IndexingParameters.cs

```diff
@@ -29,12 +25,11 @@ public partial class IndexingParameters
         /// <summary>
         /// Initializes a new instance of the IndexingParameters class.
         /// </summary>
-        public IndexingParameters(int? batchSize = default(int?), int? maxFailedItems = default(int?), int? maxFailedItemsPerBatch = default(int?), bool? base64EncodeKeys = default(bool?), IDictionary<string, object> configuration = default(IDictionary<string, object>))
+        public IndexingParameters(int? batchSize = default(int?), int? maxFailedItems = default(int?), int? maxFailedItemsPerBatch = default(int?), IDictionary<string, object> configuration = default(IDictionary<string, object>))
```

Comment:
Since we're rolling this by hand now, would it be better to set all the ctor parameter default values to null instead of verbose default(T)?